website/content/install.rst

```diff
@@ -45,6 +45,9 @@ The quickest way is to install `libenchant` using `Homebrew <https://brew.sh/>`_
     brew update
     brew install enchant
 
+If you are using `MacPorts <https://www.macports.org/>`_ you can also
+install the `enchant2` package.
```

Comment:
…install the `enchant2 <https://github.com/macports/macports-ports/tree/master/textproc/enchant2>`_ package but make sure to add the port variants for the spellers you’d like to use. For example, to build the enchant library for aspell and hunspell, use
.. code:: bash
    > sudo port install enchant2 +aspell +hunspell +applespell